From a63c314ab8d1887a6ae425fefdfe84d2d2934161 Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Thu, 21 Dec 2017 13:39:06 +0900
Subject: [PATCH 673/909] arm64: dts: r8a7795-es1-h3ulcb: Add MMNGR driver node

commit ee17cc83047083fff2d465ad2e6618ab65a59bab from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds MMNGR driver for H3ULCB board on R8A7795 ES1.x SoC.

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
index 0006f33a8d2e..7b5c6915d92f 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
@@ -61,6 +61,11 @@
 			reg = <0x00000000 0x70000000 0x0 0x10000000>;
 		};
 	};
+
+	mmngr {
+		compatible = "renesas,mmngr";
+		memory-region = <&mmp_reserved>, <&lossy_decompress>;
+	};
 };
 
 &sdhi2 {
-- 
2.17.1

